jscomp/others/belt_Option.ml

```diff
@@ -0,0 +1,27 @@
+let getExn = function
+  | Some x -> x
+  | None -> assert false
+
+let fold opt default f = match opt with
+  | Some x -> f x
+  | None -> default
+
+let map opt f = match opt with
```

Comment:
I think we'd also expose some `mapU` like other modules for consistency. But those can wait after this PR